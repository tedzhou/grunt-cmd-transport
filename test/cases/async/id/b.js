define(function(require){
	var c = require('id/f');
	return c;
})