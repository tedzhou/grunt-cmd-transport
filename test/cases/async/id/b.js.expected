define("id/b", [ "id/f", "id/e" ], function(require) {
    var c = require("id/f");
    return c;
});